src/commons/api/ca.rs

```diff
@@ -1394,18 +955,25 @@ impl fmt::Display for ParentStatuses {
                         write!(f, "LAST KNOWN Resource Entitlements:")?;
                     }
 
-                    if status.entitlements.is_empty() {
+                    if status.classes.is_empty() {
```

Comment:
Are these changes in the set of fields that are output intentional and if so why?

Reply:
Main difference is that instead of interpreting some bits we now just keep the complete response as the parent sent it. I think this will be better for debugging purposes.

It turned out that doing this was easier than changing the existing functions to use the new rpki-rs code.

In any case this is pretty much hidden, debug info.